### DISCUSSION TITLE
Parsec not starting on launch

### DISCUSSION THREAD

**Discussion by YoshiGamerFR** [Atlas-OS/Atlas#1236] [Help & Support] [answered]
I want to have Parsec started when I boot up my PC for when I am not at home to get access to it, but Atlas blocks apps launching on PC startup. Can anyone help me for that?

**Comment by kamild1996** [comment, marked as answer]
When you run the Parsec installer, you have an option to choose the installation type. Make sure you choose "Per computer". This way, Parsec operates as a system service and will start on bootup regardless of whether or not you're logged into the system.

Alternatively, open the group policy editor, go to Computer Configuration > Administrative Templates > Windows Components > Windows Logon Options and set the following:
- "Sign-in and lock last interactive user automatically after a restart" to "Enabled"
- "Configure the mode of automatically signing in and locking last interactive user after a restart or cold boot" to "Enabled" and change the drop-down setting to "Always Enabled"

This should re-enable ARSO, which is what allows your system to automatically login the last user and lock the computer on startup.

**Reply by YoshiGamerFR** [reply]
I will try this out when I get home, thank you!

**Reply by YoshiGamerFR** [reply]
I did reinstall Parsec and chose "Per computer" and it works perfectly fine now, thank you!
So I guess this was not related to Atlas OS at all like I thought it was...
